From 5d4c24bcca7f8d7f550720461373fb84eefef27e Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sat, 12 May 2018 14:16:30 +0200
Subject: [PATCH] Fix atrailers build

i2i function is used in atrailers_service.cc but this function is
defined only if defined(HAVE_JS) || defined(HAVE_TAGLIB) ||
defined(YOUTUBE) || defined(HAVE_LIBMP4V2) as a result compilation
fails if HAVE_CURL is set but HAVE_JS and HAVE_TAGLIG are not.
As youtube and libmp4v2 support have been dropped, replace those by
ATRAILERS in this list.

[Upstream Status: merged (https://github.com/gerbera/gerbera/pull/300)]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/string_converter.cc | 2 +-
 src/string_converter.h  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/string_converter.cc b/src/string_converter.cc
index b0b1f20f..79f46008 100644
--- a/src/string_converter.cc
+++ b/src/string_converter.cc
@@ -226,7 +226,7 @@ Ref<StringConverter> StringConverter::p2i()
 }
 #endif
 
-#if defined (HAVE_JS) || defined(HAVE_TAGLIB) || defined(YOUTUBE) || defined(HAVE_LIBMP4V2)
+#if defined (HAVE_JS) || defined(HAVE_TAGLIB) || defined(ATRAILERS)
 
 Ref<StringConverter> StringConverter::i2i()
 {
diff --git a/src/string_converter.h b/src/string_converter.h
index c0b64f29..ca8ae16f 100644
--- a/src/string_converter.h
+++ b/src/string_converter.h
@@ -66,7 +66,7 @@ public:
     static zmm::Ref<StringConverter> p2i();
 
 #endif
-#if defined(HAVE_JS) || defined(HAVE_TAGLIB) || defined(YOUTUBE) || defined(HAVE_LIBMP4V2)
+#if defined(HAVE_JS) || defined(HAVE_TAGLIB) || defined(ATRAILERS)
     /// \brief safeguard - internal to internal - needed to catch some
     /// scenarious where the user may have forgotten to add proper conversion
     /// in the script.
-- 
2.14.1

